cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -931,6 +962,7 @@ construct_runtime!(
 		// Bridge utilities.
 		ToWococoXcmRouter: pallet_xcm_bridge_hub_router::<Instance1>::{Pallet, Storage, Call} = 43,
 		ToRococoXcmRouter: pallet_xcm_bridge_hub_router::<Instance2>::{Pallet, Storage, Call} = 44,
+		ToWestendXcmRouter: pallet_xcm_bridge_hub_router::<Instance3>::{Pallet, Storage, Call} = 45,
```

Comment:
```suggestion
		ToRococoXcmRouter: pallet_xcm_bridge_hub_router::<ToRococoRouterInstance>::{Pallet, Storage, Call} = 44,
		ToWestendXcmRouter: pallet_xcm_bridge_hub_router::<ToWestendRouterInstance>::{Pallet, Storage, Call} = 45,
```

Reply:
actually, this does not compile:
```
error[E0412]: cannot find type `ToWococoXcmRouterInstance` in crate `pallet_xcm_bridge_hub_router`
   --> cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs:963:53
    |
963 |         ToWococoXcmRouter: pallet_xcm_bridge_hub_router::<ToWococoXcmRouterInstance>::{Pallet, Storage, Call} = 43,
    |                                                           ^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `pallet_xcm_bridge_hub_router`
```

need to check if this `construct_runtime` magic allows something like that or if it just allows instances from https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/support/src/instances.rs

Reply:
Yeah, it was never working that way, unfortunately :(